Rakefile

```diff
@@ -425,3 +425,288 @@ end
 
 spec = Gem::Specification.find_by_name 'gettext-setup'
 load "#{spec.gem_dir}/lib/tasks/gettext.rake"
+
+namespace :pwsh do
+  desc "Generate pwsh from Bolt's command line options"
+  task :generate_module do
+    @pwsh_verbs = {
+      'apply'      => 'Invoke',
+      'convert'    => 'Convert',
+      'createkeys' => 'New',
+      'init'       => 'New',
+      'install'    => 'Install',
+      'encrypt'    => 'Protect',
+      'decrypt'    => 'Unprotect',
+      'migrate'    => 'Update',
+      'run'        => 'Invoke',
+      'show'       => 'Get',
+      'upload'     => 'Send' # deploy? publish?
```

Comment:
I have no idea what's typical for powershell, but I like 'send' the most of the three :shrug:

Reply:
I have the link to the recommended verbs and nouns in the commit message, but I should probably include it here also

Reply:
Added link